server/rocksdb_container.h

```diff
@@ -0,0 +1,43 @@
+#ifndef SERVER_ROCKSDB_CONTAINER_H
+#define SERVER_ROCKSDB_CONTAINER_H
+
+#include <cstdint>
+#include <set>
+#include <string>
+
+#include "absl/strings/string_view.h"
+#include "rocksdb/db.h"
+
+namespace witnesskvs::server {
+
+class RocksDBContainer {
+  public:
+    // Creates a RocksDBContainer with num shards.
+    // Requirement: shards must be a power of 2.
```

Comment:
Why ?

Reply:
Okay, nvm understood based on GetShardId()

Reply:
We're doing bit-reversal of the string to find the shard id. Powers of two makes it easier when dealing with bits...

Reply:
commented.